aruco_detect/src/aruco_detect.cpp

```diff
@@ -341,7 +342,13 @@ void FiducialsNode::imageCallback(const sensor_msgs::ImageConstPtr & msg) {
         vector <int>  ids;
         vector <vector <Point2f> > corners, rejected;
         vector <Vec3d>  rvecs, tvecs;
-
+        bool isFisheye;
+        nh.getParam("/aruco_detect/isFisheye", isFisheye);
+        if (isFisheye)
+        {
+            cv::fisheye::undistortImage(cv_ptr->image,cv_ptr->image,cameraMatrix,distortionCoeffs,cameraMatrix,cv::Size(cv_ptr->image.cols,cv_ptr->image.rows));
```

Comment:
If you undistort the image, then you should not pass the distortion coefficients into `estimatePoseSingleMarkers`

Reply:
@jim-v thanks for the feedback. I'll fix the formatting. Regarding your comment on `estimatePoseSingleMarkers`, if the user feeds in fisheye images, and passes the distortion coefficients to the `estimatePoseSingleMarkers`, will the aruco detection work accurately, without needing to undistort the images prior to calling the function?

Reply:
I don't `estimatePoseSingleMarkers` will do a fisheye undistortion, so you'll either want to undistort the image as your PR does, or undistort the points to get a fisheye undistortion.  In this case you should make sure that the undistortion is not done twice by sending zeros to `estimatePoseSingleMarkers`.

Reply:
@jim-v Thank you! Your comments were very helpful. I will make the updates on the weekend update my PR accordingly.